docs/src/main/sphinx/release/release-466.md

```diff
@@ -0,0 +1,80 @@
+# Release 466 (27 Nov 2024)
+
+## General
+
+* Improve performance of queries with grouping on joins. ({issue}`23812`)
+* Improve client protocol throughput by introducing the spooling protocol.
+  ({issue}`24214`)
+* Improve performance of querying `system.jdbc.tables` for multiple catalogs.
+  ({issue}`24159`)
+* Add support for connectors to improve performance for `MERGE` statements by
+  only changing updated columns. ({issue}`24075`)
+* Add support for changing field types inside a map. ({issue}`24248`)
+* Replace Python-based launcher script with fully-compatible, go-based
+  executable. Removes the requirement for a Python installation on Trino cluster
+  nodes. ({issue}`24271`)
+* Improve error messages for `INTERVAL` literals. ({issue}`23002`)
+
+## Security
+
+* Add support [data access control with Apache
+  Ranger](/security/apache-ranger-access-control), including support for
+  column-masking, row-filtering, and audit logging. ({issue}`22675`)
+
+## JDBC driver
+
+* Automatically use the spooling protocol, when configured on the Trino cluster,
+  and add the parameter `encoding` to optionally set the preferred encoding from
+  the JDBC driver. ({issue}`24214`)
+* Improve performance of decompression when using Java 22 or newer as runtime
+  for the JDBC driver. ({issue}`24263`)
+
+## Docker image
+
+* Replace Python-based launcher script with fully-compatible, go-based
+  executable, and remove Python from the Docker image. ({issue}`24271`)
+
+## CLI
+
+* Automatically use the spooling protocol, when configured on the Trino cluster,
+  and add the option `--encoding` to optionally set the preferred encoding from
+  the CLI. ({issue}`24214`)
+* Improve performance of decompression when using Java 22 or newer as runtime
+  for the CLI. ({issue}`24263`)  
+
+## BigQuery connector
+
+* Add support for `LIMIT` pushdown. ({issue}`23937`)
+
+## Delta Lake connector
+
+* Improve performance of querying `system.jdbc.tables`. ({issue}`24110`)
```

Comment:
This sounds weird. How about "Improve performance **when** querying ..."

Reply:
True. The impact is actually in the JDBC client. These are tables that the client uses to obtain metadata to implement some of the APIs from JDBC.

Reply:
What JDBC client? Also in what case is the jdbc client even used .. the metadata sits in teh metastore .. its the a jdbc abstraction on top that applies to all metastore/catalogs?

Reply:
Got it now ..

Reply:
JDBC client == Trino JDBC driver. This table is used in the implementation of `java.sql.DatabaseMetaData.getTables()`